[Bug] Warning that amplitudejs should be marked as external

```
✘ [ERROR] Could not resolve "amplitudejs/dist/amplitude"

    js/player.js:1:22:
      1 │ import Amplitude from 'amplitudejs/dist/amplitude';
        ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This can be resolved by marking the file as external in our config.exs

--- a/config/config.exs
+++ b/config/config.exs
@@ -46,7 +46,8 @@ config :phoenix, :json_library, Jason
 config :esbuild,
   version: "0.14.1",
   default: [
-    args: ~w(js/app.js --bundle --target=es2016 --external:amplitudejs --external:*.webmanifest --outdir=../priv/static/assets),
+    args:
+      ~w(js/app.js --bundle --target=es2016 --external:amplitudejs --external:*.webmanifest --outdir=../priv/static/assets),
     cd: Path.expand("../assets", __DIR__),
     env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
   ]

